cgi.go

```diff
@@ -71,8 +75,14 @@ func (c *CGI) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyhttp.H
 	errorBuffer.Reset()
 	defer bufPool.Put(errorBuffer)
 
+	exePath := repl.ReplaceAll(c.Executable, "")
+	_, err := os.Stat(exePath)
+	if err != nil {
+		return c.notFound(w, r, next)
```

Comment:
directly returning `caddyhttp.Error(http.StatusNotFound, nil)` without another function call seems more straight forward - especially since the function doesn't use any of the parameters.

Reply:
I use the function here taking into account that I might consider adding the passThru option later, and when this option is true, call the next handler to handle it.